suite-native/module-device-settings/src/screens/FirmwareUpdateScreen/FirmwareChangelog.tsx

```diff
@@ -0,0 +1,80 @@
+import { useMemo } from 'react';
+import { useSelector } from 'react-redux';
+
+import { BottomSheet, Button, Text } from '@suite-native/atoms';
+import { selectFirmwareChangelog } from '@suite-common/wallet-core';
+import { Translation } from '@suite-native/intl';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+type FirmwareChangelogProps = {
+    isVisible: boolean;
+    onClose: () => void;
+};
+
+const changelogSectionTitleTextStyle = prepareNativeStyle(utils => ({
+    ...utils.typography.highlight,
+    paddingTop: utils.spacings.sp24,
+}));
+
+const buttonContainerStyle = prepareNativeStyle(utils => ({
+    marginTop: utils.spacings.sp32,
+}));
+
+const ChangelogSectionTitle = ({ children }: { children: React.ReactNode }) => {
+    const { applyStyle } = useNativeStyles();
+
+    return <Text style={applyStyle(changelogSectionTitleTextStyle)}>{children}</Text>;
+};
+
+export const FirmwareChangelog = ({ isVisible, onClose }: FirmwareChangelogProps) => {
+    const firmwareChangelog = useSelector(selectFirmwareChangelog);
+    const { applyStyle } = useNativeStyles();
+
+    const formattedChangelog = useMemo(() => {
+        if (!firmwareChangelog) {
+            return (
+                <Text>
+                    <Translation id="moduleDeviceSettings.firmware.firmwareUpdateScreen.changelog.changelogUnavailable" />
+                </Text>
+            );
+        }
+
+        if (typeof firmwareChangelog === 'string') {
+            return <Text>{firmwareChangelog}</Text>;
```

Comment:
It seems that formatting may also be used in this case.

Reply:
I went though changelogs in trezor/data repo and I haven't seen any single occurence of formatting so I won't be doing now, in future we may need to use full fledged markdown library, but for now it looks like overkill to me.